webpack.config.js

```diff
@@ -20,6 +20,7 @@ Encore
         config.useBuiltIns = 'usage';
         config.corejs = 3;
     })
+    .enableSingleRuntimeChunk()
```

Comment:
Why so?

Reply:
I allowed to avoid this error when launching the app locally : https://stackoverflow.com/a/53436451/636849

Reply:
Hum this is strange, I don't have the problem locally :/ .

Reply:
I removed it